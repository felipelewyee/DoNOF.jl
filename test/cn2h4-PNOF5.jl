using DoNOF

mol = """
0 1
  N   -1.5035843    0.2706130    0.5419655
  N   -0.5232982    0.4500715   -0.4991327
  C   -1.1167762    1.5791093    0.1337598
  H   -2.3950838    0.0155292    0.0588042
  H    0.4038793    0.2582618   -0.0565970
  H   -0.4549142    2.0832844    0.8709291
  H   -1.8723250    2.0908323   -0.5026324
"""

bset,p = DoNOF.molecule(mol,"cc-pvdz",spherical=true)

p.ipnof = 5

DoNOF.energy(bset,p)

