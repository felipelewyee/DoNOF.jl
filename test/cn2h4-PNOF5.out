Computing Integrals
Elapsed time:    1.70 Seconds

Number of basis functions                   (NBF)    = 62
Number of auxiliary basis functions         (NBFAUX) = 302
Inactive Doubly occupied orbitals up to     (NO1)    = 0
No. considered Strongly Doubly occupied MOs (NDOC)   = 12
No. considered Strongly Singly occupied MOs (NSOC)   = 0
NO. of Weakly occ. per St. Doubly occ.  MOs (NCWO)   = 4
Dimension of the Nat. Orb. subspace         (NBF5)   = 60
No. of electrons                                     = 24
    No. of alpha electrons                           = 12
    No. of beta electrons                            = 12
Multiplicity                                         = 1

Geometry
========
 N      -1.5035843       0.2706130       0.5419655
 N      -0.5232982       0.4500715      -0.4991327
 C      -1.1167762       1.5791093       0.1337598
 H      -2.3950838       0.0155292       0.0588042
 H       0.4038793       0.2582618      -0.0565970
 H      -0.4549142       2.0832844       0.8709291
 H      -1.8723250       2.0908323      -0.5026324

No violations of the orthonormality
No violations of the orthonormality
 
PNOF5/ADABelief Calculation
==================
 
  Nitext  Nit_orb  Time   Nit_occ  Time         Eelec          Etot           Ediff      maxdiff    MaxGrad_orb  Grad_occ
     1      10    9.0e+00    9    1.1e+01  -222.16798424  -145.06185539 -10221.16798424   0.268159   1.1e+00   1.8e-04 0.00
     2      10    5.5e-02   15    1.5e-02  -225.83129376  -148.72516491    -3.66330953   0.105492   4.2e-01   2.1e-04 0.01
     3      10    4.0e-02   14    1.3e-02  -226.22635363  -149.12022478    -0.39505987   0.028310   1.1e-01   2.1e-04 0.01
     4      10    4.1e-02   13    1.1e-02  -226.27442918  -149.16830033    -0.04807555   0.020103   8.0e-02   2.1e-04 0.02
     5      10    3.7e-02   12    1.1e-02  -226.29556927  -149.18944042    -0.02114009   0.018214   7.3e-02   1.9e-04 0.02
     6      10    4.3e-02    8    7.5e-03  -226.30482938  -149.19870053    -0.00926011   0.020687   8.3e-02   2.4e-04 0.02
     7      10    3.9e-02    6    5.7e-03  -226.30789766  -149.20176881    -0.00306828   0.016266   6.5e-02   1.7e-04 0.02
     8      10    5.3e-02    4    3.5e-03  -226.30861990  -149.20249105    -0.00072224   0.020893   8.4e-02   2.3e-04 0.02
     9      10    3.4e-02    0    1.7e-03  -226.30867545  -149.20254660    -0.00005555   0.020893   8.4e-02   2.3e-04 0.02
    10      16    5.3e-02    1    2.1e-03  -226.31189426  -149.20576541    -0.00321881   0.000476   1.9e-03   2.4e-04 0.03
 
RESULTS OF THE OCCUPATION OPTIMIZATION
========================================
   1    1.9999985  -15.58057704
   2    1.9999987  -15.58042610
   3    1.9999962  -11.28801960
   4    1.9834122  -0.80883240
   5    1.9832366  -0.80634335
   6    1.9780697  -0.77839746
   7    1.9780271  -0.77816879
   8    1.9745801  -0.77263110
   9    1.9780756  -0.70283458
  10    1.9780437  -0.70278822
  11    1.9912251  -0.62827811
  12    1.9910471  -0.62577221
  13    0.0072550  -0.00709388
  14    0.0071779  -0.00706457
  15    0.0157786  -0.01277126
  16    0.0155980  -0.01268927
  17    0.0246280  -0.02026613
  18    0.0177379  -0.01575313
  19    0.0176992  -0.01574277
  20    0.0152467  -0.01425738
  21    0.0150606  -0.01415529
  22    0.0000013  -0.00002468
  23    0.0000004  -0.00001009
  24    0.0000005  -0.00001324
  25    0.0007081  -0.00104908
  26    0.0005923  -0.00091033
  27    0.0027979  -0.00304265
  28    0.0028698  -0.00307773
  29    0.0006093  -0.00089232
  30    0.0019983  -0.00248998
  31    0.0020763  -0.00252879
  32    0.0011013  -0.00140797
  33    0.0010803  -0.00138677
  34    0.0000011  -0.00001915
  35    0.0000003  -0.00000342
  36    0.0000004  -0.00000108
  37    0.0005256  -0.00078211
  38    0.0005558  -0.00088694
  39    0.0017389  -0.00207713
  40    0.0017375  -0.00207283
  41    0.0001315  -0.00016301
  42    0.0011820  -0.00158429
  43    0.0012991  -0.00167901
  44    0.0002986  -0.00048306
  45    0.0003315  -0.00053165
  46    0.0000008  -0.00000107
  47    0.0000003  -0.00000027
  48    0.0000004  -0.00000017
  49    0.0004641  -0.00077700
  50    0.0004489  -0.00072466
  51    0.0016409  -0.00199316
  52    0.0017191  -0.00204118
  53    0.0000511  -0.00008241
  54    0.0010548  -0.00140795
  55    0.0008557  -0.00122578
  56    0.0001168  -0.00017822
  57    0.0001154  -0.00017719
  58    0.0000006  -0.00000161
  59    0.0000003  -0.00000005
  60    0.0000003  -0.00000023
 
----------------
 Final Energies 
----------------
Final NOF Total Energy =    -149.2057654
 
Total Orbital Internal Iterations = 106
Total Occupation Internal Iterations = 82
 
Elapsed time:   27.33 Seconds

---------------------------------
   Lowdin Population Analysis    
---------------------------------
 Idx  Atom   Population   Charge 
---------------------------------
  1     N     7.265      -0.265
  2     N     7.265      -0.265
  3     C     6.139      -0.139
  4     H     0.796       0.204
  5     H     0.795       0.205
  6     H     0.870       0.130
  7     H     0.870       0.130

---------------------------------
   M Diagnostic: 0.03
---------------------------------

